### PR TITLE
[OCPBUGS16738] Broken links to IBM docs - change zvm/71 to latest

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.10+

Issue:
-https://issues.redhat.com/browse/OCPBUGS-16738

Link to docs preview: 
- [Installing a cluster with z/VM](https://62911--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#installing-ibm-z)
- [SET SHARE link example](https://62911--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_z/installing-ibm-z.html#preferred-ibm-z-system-requirements_installing-ibm-z)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by holgwolf** [comment]
LGTM

File: installing/installing_ibm_z/installing-ibm-z.adoc [hunk 1]
```diff
@@ -54,7 +54,7 @@ include::modules/csr-management.adoc[leveloffset=+2]
 [id="additional-resources_ibmz-requirements"]
 .Additional resources
 
-* See link:https://www.ibm.com/docs/en/zvm/7.1?topic=networks-bridging-hipersockets-lan-zvm-virtual-switch[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in IBM Documentation.
+* See link:https://www.ibm.com/docs/en/zvm/latest?topic=networks-bridging-hipersockets-lan-zvm-virtual-switch[Bridging a HiperSockets LAN with a z/VM Virtual Switch] in IBM Documentation.
 
 * See link:http://public.dhe.ibm.com/software/dw/linux390/perf/zvm_hpav00.pdf[Scaling HyperPAV alias devices on Linux guests on z/VM] for performance optimization.
```
Comment (dfitzmau) [inline installing/installing_ibm_z/installing-ibm-z.adoc:59]: This link points to a site that cannot be reached. Update this link also? 

Screenshot from 2023-07-31 11-40-29

Comment (dfitzmau) [reply installing/installing_ibm_z/installing-ibm-z.adoc:59]: Hi @mburke5678 . Would you be OK to verify my peer review comments?

**Comment by dfitzmau** [comment]
Hi @SNiemann15 . Nice work on the PR. I added one comment inline. I tagged M.Burke to verify my review comments as part of the training requirement on OCP.

File: installing/installing_ibm_z/installing-ibm-z.adoc (hunk 1 above)
Comment (SNiemann15) [reply installing/installing_ibm_z/installing-ibm-z.adoc]: @dfitzmau Good catch! The publication server recently changed from http to https.

Comment (dfitzmau) [reply installing/installing_ibm_z/installing-ibm-z.adoc]: Thanks, @SNiemann15 . The link now works! Just need @mburke5678 to approve my peer review.

**Comment by mburke5678** [comment]
@dfitzmau @SNiemann15 LGTM.

**Pull request merged**